scripts/asm-installer/install_asm

```diff
@@ -2037,6 +2043,45 @@ register_gce_identity_provider() {
   retry 3 kubectl apply -f asm/identity-provider/googleidp.yaml
 }
 
+enable_service_mesh_feature() {
+  info "Enabling the service mesh feature..."
+
+  local TOKEN
+  TOKEN="$(retry 2 gcloud \
+    --project="${PROJECT_ID}" auth print-access-token)"
+
```

Comment:
It doesn't look like you use TOKEN, remove it?

Reply:
done